layouts/prelaunch.html

```diff
@@ -17,3 +17,116 @@
         <button type="submit">Skriv mig på listen!</button>
     </form>
 </div>
+<section class="pitch">
+  <div class="case container center-text">
+    <h3>Hvorfor skulle du vælge etree?</h3>
+    <p>
+      Du kunne jo også tage ud og bruge en eftermiddag på at købe et træ. Det er rigtigt nok.
+      Fordelen ved at købe et træ på nettet er at det først og fremmest er nemmere og mere overskueligt. Du
+      ved hvad du får for dine penge, og der er ikke nogen måde at blive snydt på.
+      Hvis du godt kan lide stemningen (eller vil spare leveringsomkostningerne) er det også muligt at
+      komme forbi en hyggelig afhentningsplads, hvis du køber på vores side.
```

Comment:
Denne her blok bliver meget wall-of-text'y fordi det er én paragraph. Split evt. på de linjer der er: før "Fordelen", "Hvis".